AdminUi/src/AdminUi/ClientApp/src/app/components/quotas/identity/identity-edit/identity-edit.component.ts

```diff
@@ -198,7 +200,7 @@ export class IdentityEditComponent {
 
     isAllSelected() {
         const numSelected = this.selectionQuotas.selected.length;
-        const numRows = this.identity.quotas.length;
+        const numRows = this.identity.quotas ? this.identity.quotas.filter(i => i.source === 'Individual').length : 0;
```

Comment:
Doesn't it feel bad for you to duplicate such logic (`source === "Individual"`)? If I write such code I immediately think: "What if some day there is another type of quota than cannot be deleted? Would I really remember to change all places I have this logic at?"